_posts/2023-03-03-distopia.md

```diff
@@ -0,0 +1,358 @@
+---
+layout: post
+title: Introducing Distopia
+---
+
+Calculating Euclidian distances under periodic boundary conditions is a core component in both running and analysing molecular simulations. For example, the calculation of radial distribution functions, hydrogen bonds, structure factors, and many other common analyses require calculation of one or more pairwise Euclidian distances.
```

Comment:
perhaps link to a wiki page on PBC or cite a paper ... making it more scholarly

comment on different "simulation boxes" would be helpful

Image illustrating PBC (at least in 2D) would be icing on the cake.